src/render.js

```diff
@@ -0,0 +1,46 @@
+const updateLocalStorage = require('./storage.js');
```

Comment:
- [ ] Kindly use ES6 modules. That means you should use `import/export` instead of `require` modules. It is a general requirement.

Reply:
This issue has been well addressed. Nice job! :pray: